tests/integration/tier0/destructive/method/pre_registered.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.mark.test_pre_registered_conversion
```

Comment:
We dont use the "test" in the mark names in any other conversion method so I would keep it same

Reply:
We will with the destructive test refactoring, see https://github.com/oamg/convert2rhel/pull/757/files#diff-be12858a69838593667c9cbb45eabcb670247048dbf1b22a8114208b089bced6
I agree it looks weird now, but I wanted to align with the refactor, so there are fewer changes when we rebase #757 and we don't miss something.

Reply:
Ok, no problem then